cmd/kubelet/app/server.go

```diff
@@ -783,6 +793,9 @@ func makePodSourceConfig(kc *KubeletConfig) *config.PodConfig {
 type KubeletConfig struct {
 	Address                        net.IP
 	AllowPrivileged                bool
+	Authenticator                  authenticator.Request
+	AuthorizerAttributeGetter      authorizer.RequestAttributesGetter
+	Authorizer                     authorizer.Authorizer
```

Comment:
this is the config that plumbs all the values used to construct the kubelet and kubelet auth objects